syntax = "proto3";
import "common.proto";
package das;

service AttentionBroker {
    rpc ping (Empty) returns (Ack) {}
    rpc stimulate (HandleCount) returns (Ack) {}
    rpc correlate (HandleList) returns (Ack) {}
}
